src/new-components/heroes/index.tsx

```diff
@@ -0,0 +1,2 @@
+export { ImageHero } from './image-hero';
+export { AuroraHero } from './aurora-hero';
```

Comment:
I had the feeling this makes sense for the heroes to define what is meant to be used on the outside. I'm not sure this will trigger someone of my fellow reviewers because we never did that before.